karma.conf.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+//  depMain :: String -> String
+var depMain = function(name) {
+  var pkg = require(name + '/package.json');
+  var main = 'main' in pkg ? pkg.main : 'index.js';
+  return './node_modules/' + name + '/' + main;
+};
+
+//  dependencies :: Array String
+var dependencies = Object.keys(require('./package.json').dependencies);
+
+// https://saucelabs.com/platforms
```

Comment:
If you're making other changes would you mind adding a second space after `//`? :)